### PR TITLE
Tests should always supply a commit ID

### PR DESCRIPTION
**Pull request by Naatan**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2527" title="DX-2527" target="_blank">BugDX-2527</a>  Nightly failure: lots of things that need commitIDs
  </td></table>
  <br />

### CODE REVIEW

File: test/integration/branch_int_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (suite *BranchIntegrationTestSuite) TestBranch_List() {
 	cp.Expect("To switch to another branch,")
 	cp.ExpectExitCode(0)
 
-	ts.PrepareProject("ActiveState-CLI/small-python", "")
+	ts.PrepareProject("ActiveState-CLI/small-python", "00000000-0000-0000-0000-000000000000")
```
Comment (mitchell-as) [inline test/integration/branch_int_test.go:32]: Is it worth allowing this to be empty and default to this UUID in `PrepareProject()`? Technically, this UUID does not exist for this project, so leaving it empty from a visual perspective makes sense to me.

File: test/integration/export_int_test.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ func (suite *ExportIntegrationTestSuite) TestExport_ConfigDir() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	ts.PrepareProject("cli-integration-tests/Export", "")
+	ts.PrepareProject("cli-integration-tests/Export", "00000000-0000-0000-0000-000000000000")
```
Comment (mitchell-as) [inline test/integration/export_int_test.go:68]: If we're not leaving this blank, than maybe we should just always use a valid UUID. It's weird that some tests use a valid UUID and others use this junk one.

**Review by mitchell-as — APPROVED** [review]
I'll approve this, but I'm skeptical that explicitly using a junk UUID is a good solution.

File: test/integration/export_int_test.go (hunk 2 above)
Comment (Naatan) [reply test/integration/export_int_test.go:68]: That's fair. I definitely wanted to go the explicit route.

**Review by mitchell-as — APPROVED** [review]
Perfect, thanks!

**Pull request merged**